src/puppetlabs/puppetdb/config.clj

```diff
@@ -335,13 +335,8 @@
 
 (defn add-web-routing-service-config
   [config-data]
-  (let [default-web-router-service
-        {:puppetlabs.puppetdb.cli.services/puppetdb-service "/pdb/query"
-         :puppetlabs.puppetdb.http.command/puppetdb-command-service "/pdb/cmd"
-         :puppetlabs.puppetdb.meta/metadata-service "/pdb/meta"
-         :puppetlabs.puppetdb.admin/admin-service "/pdb/admin"
-         :puppetlabs.puppetdb.dashboard/dashboard-service "/pdb"
-         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}
+  (let [default-web-router-service {:puppetlabs.puppetdb.metrics/metrics-service "/metrics"
+                                    :puppetlabs.puppetdb.pdb-routing/pdb-routing-service "/pdb"}
```

Comment:
I don't think you can do this if you want to swap in a PE version of the routing service, you'll have to do it conditionally like we do for the dashboard-redirect below